xml/storage_iscsi.xml

```diff
@@ -1434,7 +1434,7 @@ o- tpg1 .............................................. [no-gen-acls, no-auth]
       o- portals ............................................... [Portals: 1]
         o- 0.0.0.0:3260 ................................................ [OK]</screen>
    <para>
-    There is now an iSCSI target that has a 1GB file-based back-store. The
+    There is now an iSCSI target that has a 1&nbsp;GB file-based backstore. The
```

Comment:
needs a space between 1 and GB

Reply:
As before, that is what the change was **for**.

Reply:
I blame society.